internal/ciTools/bucket.go

```diff
@@ -0,0 +1,21 @@
+package ciTools
+
+import (
+	"fmt"
+
+	"github.com/kubefirst/kubefirst/internal/aws"
+	"github.com/spf13/viper"
+)
+
+func CreateBucket() (string, error) {
+	randomName := viper.GetString("bucket.rand")
+	bucket := "ci-state"
+	bucketName := fmt.Sprintf("k1-%s-%s", bucket, randomName)
+	aws.CreateBucket(false, bucketName)
+
+	viper.Set(fmt.Sprintf("bucket.%s.created", bucket), true)
+	viper.Set(fmt.Sprintf("bucket.%s.name", bucket), bucketName)
+	viper.WriteConfig()
```

Comment:
Test on `clean` if the bucket is removed as desired.

Reply:
included bucket destroy in CI action remove